mysql/tests/test_mysql.py

```diff
@@ -70,9 +70,11 @@ def test_minimal_config(aggregator, dd_run_check, instance_basic):
 def test_complex_config(aggregator, dd_run_check, instance_complex):
     mysql_check = MySql(common.CHECK_NAME, {}, [instance_complex])
     dd_run_check(mysql_check)
+
     _assert_complex_config(
         aggregator,
-        [tags.DATABASE_INSTANCE_RESOURCE_TAG.format(hostname='stubbed.hostname')],
+        tags.SC_TAGS + [tags.DATABASE_INSTANCE_RESOURCE_TAG.format(hostname='stubbed.hostname')],
+        tags.METRIC_TAGS_WITH_RESOURCE,
```

Comment:
it looks like this is asserting that the `dd.resource.database_instance:*` would be in this array twice? I am assuming the test case normalizes the tag set somewhere and not that the tags would show up twice?

Reply:
Nah, the array only includes the internal resource tag once. `tags.SC_TAGS` omits the internal tag because it had to be a formatted string depending on the hostname.

Reply:
It does but one of those parameters is the service check tags and the other is the metric tags. They had to be separated because service check tags will include the internal resource tag even in the end to end case while metrics won't have the internal tags (because it gets transformed into actual resources on metrics).